docusaurus.config.js

```diff
@@ -218,6 +209,21 @@ module.exports = {
       },
     ],
   ],
+  themes: [
```

Comment:
why themes instead of plugins?

Reply:
Thats the new way its documented - docusaurus core team suggested switching

https://github.com/easyops-cn/docusaurus-search-local?tab=readme-ov-file#usage